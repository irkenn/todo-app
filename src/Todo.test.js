import { render } from '@testing-library/react';
import ToDo from './Todo';

it("renders without crashing", 
  function(){
    render(<ToDo/>);
  });

it("matches the snapshot", () => {
  const { asFragment } = render(<ToDo/>);
  expect(asFragment()).toMatchSnapshot()
});