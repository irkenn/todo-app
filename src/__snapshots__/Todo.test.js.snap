// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="todo-wrapper"
  >
    <div
      class="todo"
    />
    <button
      class="delete-todo-button"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
